src/08_ecosystem/00_chapter.md

```diff
@@ -0,0 +1,51 @@
+# The Async Ecosystem
+Most Rust asynchronous projects will need third-party crates to function.
```

Comment:
nit: Maybe crates provided by the community? "third-party" is used a lot at companies who normally roll their own "first-party" stuff, but others may not use this term.